PurchasesCoreSwift/Public/Offering.swift

```diff
@@ -113,7 +113,7 @@ import Foundation
     }
 
     /// :nodoc:
-    @objc public func object(forKeyedSubscript key: String) -> Package? {
```

Comment:
Turns out, we need the OTHER subscript method for subscripting to work in Swift.